CONTRIBUTING.md

```diff
@@ -0,0 +1,43 @@
+# Contributing Guidelines
+
+Cycloid Team is glad to see you contributing to this project ! In this document, we will you provide some guidelines in order to help get your contribution accepted.
+
+## Reporting an issue
+
+When you find a bug in Terracognita, it should be reported using [GitHub issues](https://github.com/cycloidio/terracognita/issues). Please define key information like your Operating System (OS), Terracognita origin (docker or from source) and finally the version you are using.
```

Comment:
Could we create some templates for that maybe?

Reply:
A minimalist template could be nice (people do not fill big templates :smiley: ):

- OS
- build from source or use docker image
- version
- ?

Reply:
Yes just general information, for now this 3 would be ok.